src/types.rs

```diff
@@ -1,15 +1,5 @@
-use ruby_sys::types;
+pub use ruby_sys::types::*;
```

Comment:
I don't remember offhand if it's OK (idiomatically speaking) to re-export with a `*` in Rust.

*(Apart from this, I don't have any problems with this PR.)*